starky/src/serializer.rs

```diff
@@ -829,20 +831,23 @@ mod tests {
         assert_eq!(actual, starkproof);
     }
 
-    // TODO: test bls12381
+    #[test]
     fn test_serialize_stark_proof_bls12381_ser_der() {
+        env_logger::try_init().unwrap_or_default();
         let mut pil = load_json::<PIL>("data/fib.pil.json").unwrap();
         let mut const_pol = PolsArray::new(&pil, PolKind::Constant);
         const_pol.load("data/fib.const").unwrap();
         let mut cm_pol = PolsArray::new(&pil, PolKind::Commit);
         cm_pol.load("data/fib.cm").unwrap();
-        let stark_struct = load_json::<StarkStruct>("data/starkStruct.json").unwrap();
+        let stark_struct =
+            load_json::<StarkStruct>("data/final.starkStruct.bls12381.json").unwrap();
```

Comment:
just cp the starkStruct.json, and change the hash type to BLS12381.  Don't use the final..., it has different `nBits` from what we need.

Reply:
It's down.